src/Pages/Install.cshtml

```diff
@@ -1,268 +1,273 @@
-            <div class="form-check form-switch">
-                <input class="form-check-input" type="checkbox" id="downloadmodel_sdxl1refiner">
-                <label class="form-check-label" for="downloadmodel_sdxl1refiner">Stable Diffusion XL 1.0 (Refiner)</label>
-                <div class="install_hint">Bonus model, can be added onto SDXL 1.0 Base to refine the small details of your output image to truly top-tier quality, at the cost of requiring more VRAM and time to generate. (Note: does not work well on its own, expects to be used as a Refiner over outputs from the Base).</div>
-            </div>
-        </fieldset>
-        <label>Or community models!</label>
-        <div class="install_hint">You can download an endless variety of community-made models from online sites such as <a href="https://huggingface.co/spaces/huggingface-projects/diffusers-gallery" target="_blank" rel="noopener noreferrer">HuggingFace</a> or <a href="https://civitai.com/" target="_blank" rel="noopener noreferrer">Civitai</a>. (NOTICE: These are examples of third-party websites featuring user-submitted models, Stability.AI offers no guarantees about what content may exist there, browse with caution.) Remember to only download '.safetensors' files, never '.ckpt' ones.</div>
-    </div>
-
-    <div id="installer_section_end" style="display: none;">
-        You have selected:
-        <br>&bullet; Theme: <span class="final_confirm_info" id="theme_val_info"></span>
-        <br>&bullet; UI Is For: <span class="final_confirm_info" id="installed_for_val_info"></span>
-        <br>&bullet; Backend: <span class="final_confirm_info" id="backend_val_info"></span>
-        <br>&bullet; Model: <span class="final_confirm_info" id="model_val_info"></span>
-        <br>Are you sure this is all correct?
-        <div class="install_hint">This is your last chance to go back and change before installing. If you want to change your choices later, there's a settings menu available within StableSwarmUI, and models can be downloaded any time you wish and put in the models folder.</div>
-        <br><button id="installer_button_confirm" class="btn btn-primary basic-button" autocomplete="off">Yes, I'm sure (Install Now)</button>
-        <div class="install_output" id="install_output"></div>
-        <div class="install_output" id="install_progress_spot"></div>
-        <div class="install_progress_bar" id="install_progress_bar"></div>
-        <div class="install_progress_step_bar" id="install_progress_step_bar"></div>
-    </div>
-    <hr>
-    <div class="section_buttons">
-        <span id="bottom_info"></span>
-        <button id="installer_button_back" class="btn btn-primary basic-button" disabled autocomplete="off">Back</button>
-        <button id="installer_button_next" class="btn btn-primary basic-button" autocomplete="off">Agree</button>
-    </div>
-</div>
-
-@section Scripts {
-    <script src="/js/installer.js?vary=@Utilities.VaryID"></script>
-}
+@page

```

Comment:
heck happened with the commit on this file?

Reply:
For some reason github constantly thinks that I completely rewrote and changed the entirety of the file, even though I only added one section and changed nothing. I've never been able to figure out how to fix it, but it happens on every file I upload if I don't change it within githubs editor, even if I copy-paste it.

Reply:
Ah, yeah, probably works better if you actually use git locally rather than the website -- uh probably for this PR just remove the changes to the installer files and I'll reapply the update after

Reply:
Alright, will-do! Thank you

Reply:
> For some reason github constantly thinks that I completely rewrote and changed the entirety of the file, even though I only added one section and changed nothing. I've never been able to figure out how to fix it, but it happens on every file I upload if I don't change it within githubs editor, even if I copy-paste it.

line endings? encoding? People are constantly borking stuff by somehow setting file modifications to "UTF-8 BOM" vs "UTF-8" at my day job.